README.md

```diff
@@ -0,0 +1,16 @@
+# Static Web Apps - Mongoose starter
+
+This project is designed to be a starter [Azure Static Web Apps](https://docs.microsoft.com/en-us/azure/static-web-apps/overview) with [Mongoose](https://mongoosejs.com/). It uses the following resources:
```

Comment:
Removing the hard-coded locale string so readers will see this content in their default language.

```suggestion
This project is designed to be a starter [Azure Static Web Apps](https://docs.microsoft.com/azure/static-web-apps/overview) with [Mongoose](https://mongoosejs.com/). It uses the following resources:
```